src/components/learner-credit-management/cards/data/utils.js

```diff
@@ -164,15 +165,19 @@ export const isInviteEmailAddressesInputValueValid = ({ learnerEmails }) => {
       validationError.message = `${invalidEmails[0]} is not a valid email.`;
     }
   } else if (duplicateEmails.length > 0) {
+    let message = `${duplicateEmails[0]} was entered more than once.`;
+    if (duplicateEmails.length > 1) {
```

Comment:
nit:  There's a variable `duplicateEmailsCount` at line 147 that can be used here
```suggestion
    if (duplicateEmailsCount > 1) {
```